allensdk/brain_observatory/behavior/behavior_ophys_experiment.py

```diff
@@ -2,19 +2,44 @@
 import pandas as pd
```

Comment:
What changed in this file other than the linting? Why did you need to touch this file?

Reply:
I think it's when I rebased my PR onto the updated RC branch, there were merge conflicts that I needed to resolve which marked the file as being touched, and my script to run black on changed files formatted it.